plugin.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "4.0.10",
```

Comment:
Do not change yet

Reply:
This was also used for testing. Changed back to 4.0.10.